packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -5639,6 +5642,16 @@ describe('plugin-meetings', () => {
             });
           });
 
+          it('should reconnect successfully if reconnectionManager.cleanUp is called before reconnection attempt', async () => {
```

Comment:
I tried to write a test that checked that validate() was called twice, but each time validate is called it's a different instance of ReconnectionManager and the functions in ReconnectionManager are no longer stubbed once it is overwritten with a new instance so it wasn't as simple as just checking ```assert.calledTwice(meeting.reconnectionManager.validate)```.